packages/grafana-runtime/package.json

```diff
@@ -57,15 +56,13 @@
     "@testing-library/user-event": "14.4.3",
     "@types/angular": "1.8.4",
     "@types/history": "4.7.11",
-    "@types/jest": "^29.2.2",
+    "@types/jest": "28.1.6",
```

Comment:
Why do you need to downgrade `jest`?

Reply:
argh, this one was from the older PR and after merging with main it probably overwrote this 😓 – will revert